modules/nw-enabling-multicast.adoc

```diff
@@ -24,15 +24,111 @@ You can enable multicast between Pods for your project.
 
 .Procedure
 
-* Run the following command to enable multicast for a project:
+* Run the following command to enable multicast for a project. Replace `<namespace>` with the namespace for the project you want to enable multicast for.
 +
 [subs="attributes+"]
 ----
-$ oc annotate {namespace} <namespace> \ <1>
+$ oc annotate {namespace} <namespace> \
     {annotation}
 ----
+
+.Verification steps
+
+To verify that multicast is enabled for a project, complete the following procedure:
+
+. Change your current project to the project that you enabled multicast for. Replace `<project>` with the project name.
++
+[source,terminal]
+----
+$ oc project <project>
+----
+
+. Create a Pod to act as a multicast receiver:
++
+[source,terminal]
+----
+$ cat <<EOF| oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mlistener
+  labels:
+    app: multicast-verify
+spec:
+  containers:
+    - name: mlistener
+      image: registry.access.redhat.com/ubi8
+      command: ["/bin/sh", "-c"]
+      args:
+        ["dnf -y install socat hostname && sleep inf"]
+      ports:
+        - containerPort: 30102
+          name: mlistener
+          protocol: UDP
+EOF
+----
+
+. Create a Pod to act as a multicast sender:
 +
-<1> The `namespace` for the project you want to enable multicast for.
+[source,terminal]
+----
+$ cat <<EOF| oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: msender
+  labels:
+    app: multicast-verify
+spec:
+  containers:
+    - name: msender
+      image: registry.access.redhat.com/ubi8
+      command: ["/bin/sh", "-c"]
+      args:
+        ["dnf -y install socat && sleep inf"]
+EOF
+----
+
+. Start the multicast listener:
```

Comment:
Step 4's lead-in sentence here ends with a colon before the substeps, while step 5's lead-in ends with a period. Suggest syncing up on one or the other (personally I use the latter).